### PR TITLE
Orderbook tutorial

### PR DESCRIPTION
**Pull request by WRRicht3r**
Polkaswap tutorial for orderbooks

### CODE REVIEW

File: src/orderbook-trading.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+![](.gitbook/assets/polkaswap-orderbook-tutorial-hero.png)
+
+## Theory
+
+An order book is a fundamental component of a decentralized exchange (DEX) that facilitates trading digital assets between buyers and sellers in a transparent and efficient manner. It serves as a virtual meta ledger that records all the buy and sell orders placed by participants in the market.
+
+This tutorial will walk you through the essential steps for trading on Polkaswap, focusing on connecting your wallet, understanding the interface, and executing trades. Let's get started.
+
+:::info
```
Comment (outoftardis) [inline src/orderbook-trading.md]:
```suggestion
::: info
```

File: src/orderbook-trading.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+For advanced Orderbook functionality as well as how to integrate the SORA Orderbook in your dApp, refer to the [Orderbooks](orderbooks.md) section.
```
Comment (outoftardis) [inline src/orderbook-trading.md]:
```suggestion
For advanced Orderbook functionality as well as for detail on how to integrate the SORA Orderbook in your dApp, refer to the [Orderbooks](orderbooks.md) section.
```

File: src/orderbook-trading.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+![](.gitbook/assets/polkaswap-orderbook-interface.png)
+
+### Explore the Interface:
+
+You'll see several key components in the Orderbook interface:
+
+- **The Order Book Widget:** Shows all the aggregated orders for a selected pair.
+- **Charts:** Visual representations of trading activity.
+- **Buy and Sell Widgets:** Where you'll place your orders.
```
Comment (outoftardis) [inline src/orderbook-trading.md]:
```suggestion
- **Charts:** Shows visual representations of trading activity.
- **Buy and Sell Widgets:** Shows where you'll place your orders.
```

File: src/orderbook-trading.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+- **Limit Order:** Allows you to set any desired price. Orders default to one month and will be automatically canceled after this period.
+
+![](.gitbook/assets/polkaswap-orderbook-limit.png)
+
+4. **Order Cancellation:** You can cancel your order at any time
+   without any fees.
+
+5. **Fees on Order Placement:** You will incur a fee when placing an
+   order. The fee amount is dependent on the price, similar to current
```
Comment (outoftardis) [inline src/orderbook-trading.md]:
```suggestion
   order. The fee amount depends on the price, similar to current
```

File: src/orderbook-trading.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+- **Market Order:** Executes at the latest pair price, similar to a swap.
+
+![](.gitbook/assets/polkaswap-orderbook-market.png)
+
+- **Limit Order:** Allows you to set any desired price. Orders default to one month and will be automatically canceled after this period.
+
+![](.gitbook/assets/polkaswap-orderbook-limit.png)
+
+4. **Order Cancellation:** You can cancel your order at any time
```
Comment (outoftardis) [inline src/orderbook-trading.md:55]: why this list starts at 4?

**Review by outoftardis — COMMENTED** [review]
@bgodlin  please address these comments in your pr

File: src/orderbook-trading.md (hunk 5 above)
Comment (WRRicht3r) [reply src/orderbook-trading.md:55]: its a continuation of the ongoing list at the beginning of the document

**Pull request closed without merging**